src/Card/CardImageCap.jsx

```diff
@@ -45,26 +46,37 @@ const CardImageCap = React.forwardRef(({
     );
   }
 
-  const handleSrcFallback = (event, altSrc) => {
+  const handleSrcFallback = (event, altSrc, imageKey) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
-      currentTarget.src = altSrc;
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      if (imageKey === 'imageCap') {
+        currentTarget.src = cardSrcFallbackImg;
+      } else {
+        setShow(s => ({ ...s, logoCap: false }));
+      }
+
+      return;
     }
+
+    currentTarget.src = altSrc;
   };
 
   return (
     <div className={classNames(className, wrapperClassName)} ref={ref}>
       <img
-        className="pgn__card-image-cap"
+        className={classNames('pgn__card-image-cap', { show: show.imageCap })}
         src={src}
-        onError={(event) => handleSrcFallback(event, fallbackSrc)}
+        onError={(event) => handleSrcFallback(event, fallbackSrc, 'imageCap')}
+        onLoad={() => setShow(s => ({ ...s, imageCap: !!src }))}
```

Comment:
[curious/question] If `src` is an optional prop for `Card.ImageCap`, should we render the `<img>` element conditionally similar to `!!logoSrc && (...)` below?

Reply:
Yes, let's do this